packages/xo-server/src/xo-mixins/metadata-backups.js

```diff
@@ -879,6 +881,9 @@ export default class metadataBackup {
 
           common.taskId = localTaskId
           logger.notice(message, common)
+          if (isRootTask) {
+            this._runningMetadataRestores.delete(localTaskId)
```

Comment:
What if something wrong occurs and we never get this task?

Why not simply a big `try`/`catch` around the method (both proxied and local)?